paramak/parametric_components/toroidal_field_coil_princeton_d.py

```diff
@@ -37,6 +38,7 @@ def __init__(
         thickness,
         distance,
         number_of_coils,
+        rotation_angle=360,
```

Comment:
This is maybe not needed since 360.0 is already the default value at RotateMixedShape level